authors/settings.py

```diff
@@ -43,13 +43,11 @@
     'cloudinary',
     'corsheaders',
     'django_extensions',
-    'rest_framework_swagger',
     'rest_framework',
-
     'authors.apps.authentication',
     'authors.apps.core',
     'authors.apps.profiles',
-
+    'rest_framework_swagger',
```

Comment:
this should be under the third party applications section along with cloudinary and the rest